docs/book/features/volume_health.md

```diff
@@ -0,0 +1,81 @@
+# vSphere CSI Driver - Volume Health
+Volume Health feature provides the health status of persistent volume. Currently, Volume Health is only supported for vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster and only supported for block volume.
```

Comment:
Can we rephrase it as

Volume Health feature provides the health status of persistent volume. Volume Health is supported for Block Volumes in vSphere with Kubernetes Cluster and Tanzu Kubernetes Grid Cluster - vSphere 7.0u1